libraries/helpers_network.rb

```diff
@@ -92,7 +92,6 @@ def consolidate_ipam(subnets, ranges, gateways, auxaddrs)
             data[s]['Gateway'] = g
             match = true
           end
-          raise "no matching subnet for gateway #{s}" unless match
```

Comment:
the inspect looks like:

```json

 "IPAM": {
            "Driver": "default",
            "Options": null,
            "Config": [
                {
                    "Subnet": "172.24.0.0/16",
                    "Gateway": "172.24.0.1"
                },
                {
                    "Subnet": "fd00:dead:beef::/48"
                }
            ]
        },
```
Docker automatically adds the IPv4 part (Gateway set) but the IPv6 subnet never matches a Gateway.